src/training/day5/Person.java

```diff
@@ -0,0 +1,37 @@
+package training.day5;
+
+public class Person {
+    private String name;
+    private String gender;
+    private int ID;
+
+    public Person(String name, String gender, int ID) {
+        this.name = name;
+        this.gender = gender;
+        this.ID = ID;
+    }
+    public String getName() {
+        return name;
+
```

Comment:
Remove this line since the method holds the logic on one line. And everywhere else.